opm/simulators/flow/LogOutputHelper.cpp

```diff
@@ -580,32 +606,107 @@ outputCumulativeReportRecord_(const std::vector<Scalar>& wellCum,
 template <typename Scalar>
 void LogOutputHelper<Scalar>::beginInjectionReport_() const
 {
-    const auto unitType = this->eclState_.getUnits().getType();
+    const auto& units = this->eclState_.getUnits();
+    const auto widths = std::array{8, 11, 6, 6, 6, 11, 11, 11, 11, 8, 8};
+
+    using namespace std::string_view_literals;
+
+    auto oil_rate =
+        [&units]()
```

Comment:
Might as well capture `units.getType()` here, e.g.,
```C++
auto oil_rate = [utype = units.getType()]()
{
    switch (utype) {
    // ...
    }
};
```
since you're not really using a lot of the `units` object internally.

Reply:
sure.